webpack.prod.config.js

```diff
@@ -75,26 +74,30 @@ Object.assign(config, {
     'openassessment-editor-textarea': path.resolve(process.cwd(), 'openassessment/xblock/static/js/src/lms/editors/oa_editor_textarea.js'),
     'openassessment-editor-tinymce': path.resolve(process.cwd(), 'openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js'),
   },
-  output: {
-    path: path.resolve(process.cwd(), 'openassessment/xblock/static/dist'),
-  },
   optimization: {},
   plugins: [
+    // Cleans the dist directory before each build
+    new CleanWebpackPlugin(),
     new Dotenv({
       path: path.resolve(process.cwd(), '.env'),
       systemvars: true,
     }),
     new MiniCssExtractPlugin({
-      filename: '[name].css',
+      filename: '[name].[chunkhash].css',
     }),
     new webpack.ProvidePlugin({
       Backgrid: path.resolve(path.join(__dirname, 'openassessment/xblock/static/js/lib/backgrid/backgrid'))
     }),
+    new WebpackManifestPlugin({
+      seed: {
+        is_dev_server: process.env.WEBPACK_DEV_SERVER || false,
+        base_url: '/static/dist',
+      }
+    })
   ],
 });
 
-config.resolve.modules = ['node_modules'].concat(
-  path.resolve(__dirname, 'openassessment/xblock/static/js/src'),
-);
+config.resolve.modules = ['node_modules', path.resolve(__dirname, 'openassessment/xblock/static/js/src')]
+config.output.path = path.resolve(process.cwd(), 'openassessment/xblock/static/dist')
```

Comment:
eslint errors in this file (missing commas/semicolons, and wrong quotes). not all of these complaints are new w/ this change.